cmd/lakectl/cmd/docs.go

```diff
@@ -48,6 +48,9 @@ This will setup a ` + "`" + `$HOME/.lakectl.yaml` + "`" + ` file with the creden
 When setting up a new installation and creating initial credentials (see [Quick start](../quickstart/index.md)), the UI
 will provide a link to download a preconfigured configuration file for you.
 
+` + "`" + `lakectl` + "`" + ` configuration items can each be controlled by an environment variable. The variable name will have a prefix of
```

Comment:
suggestion, it will have the same output
replace all: "`" + `lakectl` + "`"
with: "`lakectl`"

Reply:
done here and in this text block